docs: misc academy docs fixes (#2536)
apify/apify-docs: sources/academy/webscraping/scraping_basics_legacy/challenge/scraping_amazon.md

```

Great! But where do we go from here? We need to visit the product page and scrape the offer from there.

:::note Why only one offer per product?

> Amazon product pages list a single featured offer in their static HTML. The full list of sellers is loaded separately by JavaScript after the page loads, which means CheerioCrawler (which only fetches the raw HTML, without running JavaScript) cannot see it.

Reviewer: ```suggestion
Amazon product pages list a single featured offer in their static HTML. The full list of sellers is loaded separately by JavaScript after the page loads. CheerioCrawler can't see the list because it only fetches the raw HTML, without running JavaScript.
```

> If you need all seller offers, you would have to use [PlaywrightCrawler](../../puppeteer_playwright/index.md), which runs a real browser and can wait for that content to load. For this course, scraping the featured offer is enough to cover the key concepts.

Reviewer: ```suggestion
If you need all seller offers, you have to use [PlaywrightCrawler](../../puppeteer_playwright/index.md), which runs a real browser and can wait for that content to load. For this course, scraping the featured offer is enough to cover the key concepts.
```
